### PR TITLE
first commit. Developped a map module for the project with leaflet. F…

### PR DESCRIPTION
**Pull request by QuentinVerwaerde**
Using Leaflet, I put a map in my component
Fetched a public API that does not ask for keys to be used, in order to retrieve a usable array with GPS coordinate in it
Had to put together a new array with pictures of my cat into it and a generic Paris picture
Used map() to generate markers and popups using the two arrays

### CODE REVIEW

File: frontend/src/App.jsx [hunk 1]
```diff
@@ -1,10 +1,15 @@
 import "./App.css";
+import "./style.css";
+import "leaflet/dist/leaflet.css";
+import React from "react";
```
Comment (ioayoub) [inline frontend/src/App.jsx:4]: React import is not needed here

File: frontend/src/components/Markers.jsx [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+import "leaflet/dist/leaflet.css";
+
+function Markers() {
+  // créaton de marqueurs stylisés
+  const customIcon = new Icon({
+    iconUrl: "./src/assets/pattecoeur.png",
+    iconSize: [38, 38],
+  });
+  // création d'un tableau pour y stocker les images des popups de la carte
+  const popUp = [
```
Comment (ioayoub) [inline frontend/src/components/Markers.jsx:14]: Please move these datas in a specific folder (ex: data)

File: frontend/src/components/Markers.jsx [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+                <Popup>
+                  <img
+                    src={popUp.at(index).imageAnat}
+                    alt="imageAnat"
+                    width="200px"
+                    height="200px"
+                  />
+                </Popup>
+              </Marker>
+              //   <span key={d.c_ar}>
```
Comment (ioayoub) [inline frontend/src/components/Markers.jsx:157]: Dead code is not allowed.

File: frontend/src/App.jsx [hunk 4]
```diff
@@ -1,10 +1,15 @@
...
+import Markers from "./components/Markers";
 import Navbar from "./components/navbar/Navbar";
 
 function App() {
   return (
-    <div className="container">
-      <Navbar />
+    <div className="container"> 
+    <Navbar />
```
Comment (ioayoub) [inline frontend/src/App.jsx:11]: Fix your indentation here

**Review by ioayoub — CHANGES_REQUESTED** [review]

File: frontend/src/components/Markers.jsx [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+              //     <p>
+              //       {index}------
+              //       {/* <img src="{popUp[index].imageAnat}" alt="imageAnat" /> */}
+              //       <img src={popUp[index].imageArr} alt="imageArr" />
```
Comment (sdiakhate) [inline frontend/src/components/Markers.jsx:161]: can you remove the commented code please

**Review by sdiakhate — DISMISSED** [review]

File: frontend/src/components/Markers.jsx (hunk 2 above)
Comment (QuentinVerwaerde) [reply frontend/src/components/Markers.jsx]: Fixed it Ayoub, it's gone :)

**Review by ioayoub — APPROVED** [review]

**Review by sdiakhate — APPROVED** [review]

**Pull request merged**